src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -844,6 +867,54 @@ def _get_kfp_ui_daemon_port(self) -> int:
             port = networking_utils.find_available_port()
         return port
 
+    def _get_session_cookie(self, username: str, password: str) -> str:
+        session = requests.Session()
+        if (
+            self.config.kubeflow_hostname is None
```

Comment:
Are we 100% sure that this needs to be the case and can't be different in some custom deployments? We already log a warning in the config when setting the hostname, not sure it's necessary to always fail here?

Reply:
You're right. I replaced this with just the check that the hostname needs to exist